cmd/devtool/devtool.go

```diff
@@ -331,9 +334,9 @@ func remoteConsole(destAccountAddr string) error {
 	}
 	defer console.Stop(false)
 
-	if !controllerAddrOverride {
+	if !ethControllerOverride {
 		// f9a6cf519167d81bc5cb3d26c60c0c9a19704aa908c148e82a861b570f4cd2d7 - SetContractInfo event
-		getControllerAddressScript := `
+		getethControllerScript := `
```

Comment:
`getethControllerScript` -> `getEthControllerScript`? Same comment applies for the other appearances of this variable in this file.

Reply:
Fixed